.devcontainer/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# arm + amd compatible Dockerfile
+FROM ghcr.io/findy-network/findy-base:indy-1.16.ubuntu-18.04 AS indy-base
+
+FROM ubuntu:18.04
+
+# install indy deps and files from base
+RUN apt-get update && apt-get install -y libsodium23 libssl1.1 libzmq5 git zsh
+
+COPY --from=indy-base /usr/include/indy /usr/include/indy
+COPY --from=indy-base /usr/lib/libindy.a /usr/lib/libindy.a
+COPY --from=indy-base /usr/lib/libindy.so /usr/lib/libindy.so
+
+RUN apt-get install -y curl python3 build-essential ca-certificates && \
+  curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash && \
+  export NVM_DIR="$HOME/.nvm" && \
+  [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" && \
+  nvm install v16 && \
+  npm install yarn -g
+
+RUN sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
```

Comment:
Is there a specific reason to install oh my zsh? I think it would be good to keep the container unopinionated and just use the default shell environment.

Reply:
When you try the container it starts the default shell 'bash'. You have only seen this when you typed `zsh` to run an alternative shell w/ ohmyzsh. In any case, I removed `ohmyzsh` to move the PR along.